src/profile/data/sagas.test.js

```diff
@@ -96,16 +99,17 @@ describe('RootSaga', () => {
       const action = profileActions.fetchProfile('booyah');
       const gen = handleFetchProfile(action);
 
-      const result = [{}, [1, 2, 3]];
+      const result = [{}, [1, 2, 3], []];
```

Comment:
instead of adding an empty country list, please add 2 countries to test the scenarios.